x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/connector/60_connector_update_filtering.yml

```diff
@@ -191,16 +191,29 @@ setup:
 ---
 "Update Connector Filtering with value literal - Empty rules":
   - do:
-      catch: "bad_request"
       connector.update_filtering:
         connector_id: test-connector
         body:
           rules: [ ]
 
+  - match: { result: updated }
+
+  - do:
+      connector.get:
+        connector_id: test-connector
+
+  - match: { filtering.0.draft.rules.0.id: "DEFAULT" }
+  - match: { filtering.0.draft.rules.0.policy: "include" }
+  - match: { filtering.0.draft.rules.0.rule: "regex" }
+  - match: { filtering.0.draft.rules.0.value: ".*" }
+  - match: { filtering.0.draft.rules.0.field: "_" }
+  - match: { filtering.0.draft.rules.0.order: 0 }
+
+
+
 ---
 "Update Connector Filtering with value literal - Default rule not present":
```

Comment:
can we add another test to check what happens when we:
- only pass default rule in the `rules` 
- pass some rules + default rule in the `rules` request body (backward compatibility)